### PR TITLE
fix(docs): restore images within links

### PR DESCRIPTION
**Pull request by PowerKiKi**

### CODE REVIEW

File: docs/documentation/fxFlex-API.md [hunk 1]
```diff
@@ -182,12 +184,12 @@ Developers should consult the **[Known Issues](http://caniuse.com/#feat=flexbox)
 [Resources](http://caniuse.com/#feat=flexbox) sections.
 
 <a href="http://caniuse.com/#feat=flexbox" target="_blank">
-![caniuseflexbox](https://cloud.githubusercontent.com/assets/210413/21288118/917e3faa-c440-11e6-9b08-28aff590c7ae.png)
+  <img alt="caniuseflexbox" src="https://cloud.githubusercontent.com/assets/210413/21288118/917e3faa-c440-11e6-9b08-28aff590c7ae.png">
```
Comment (CaerusKaru) [inline docs/documentation/fxFlex-API.md]: This should be `[caniuse](http://caniuse.com/#feat=flexbox)` with no `<a>` tags

File: docs/documentation/fxFlex-API.md [hunk 2]
```diff
@@ -35,13 +35,15 @@ or
 ```
 
 * The **long-form** enables the developer to specify the grow, shrink, and basis values inline.
+
```
Comment (CaerusKaru) [inline docs/documentation/fxFlex-API.md]: Remove

File: docs/documentation/fxFlex-API.md [hunk 3]
```diff
@@ -35,13 +35,15 @@ or
...
   * `fxFlex="1 1 52%"`
   * `fxFlex="3 3 calc(15em + 20px)"`
   * `fxFlex="1 1 auto"`
 
 
 * The **short-form** enables developers to specify only the **flex-basis** and uses defaults for the shrink and grow 
 options: (default values == 1).
+
```
Comment (CaerusKaru) [inline docs/documentation/fxFlex-API.md]: Remove

File: docs/documentation/fxFlex-API.md [hunk 4]
```diff
@@ -82,13 +84,13 @@ Shown below is an example of a non-trivial layout using various **fxFlex** optio
...
 <a href="https://github.com/angular/flex-layout/blob/master/src/demo-app/app/stack-overflow/columnSpan.demo.ts#L23" target="_blank">
-![screen shot 2016-12-16 at 1 05 45 pm](https://cloud.githubusercontent.com/assets/210413/21274996/6b640f8a-c390-11e6-87ac-ca85eb6c3983.png)
+<img alt="screen shot 2016-12-16 at 1 05 45 pm" src="https://cloud.githubusercontent.com/assets/210413/21274996/6b640f8a-c390-11e6-87ac-ca85eb6c3983.png">
 </a>
 
 #### Live Demo:
 
 <a href="https://tburleson-layouts-demos.firebaseapp.com/#/stackoverflow" target="_blank">
-![screen shot 2016-12-16 at 1 00 51 pm](https://cloud.githubusercontent.com/assets/210413/21274826/bc8553f2-c38f-11e6-8188-bc7fd36026c2.png)
+  <img alt="screen shot 2016-12-16 at 1 00 51 pm" src="https://cloud.githubusercontent.com/assets/210413/21274826/bc8553f2-c38f-11e6-8188-bc7fd36026c2.png">
```
Comment (CaerusKaru) [inline docs/documentation/fxFlex-API.md]: This should be `[screen shot 2016-12-16 at 1 00 51 pm](https://tburleson-layouts-demos.firebaseapp.com/#/stackoverflow)` with no `<a>` tags

File: docs/documentation/fxFlex-API.md [hunk 5]
```diff
@@ -82,13 +84,13 @@ Shown below is an example of a non-trivial layout using various **fxFlex** optio
 #### Source Code:
 
 <a href="https://github.com/angular/flex-layout/blob/master/src/demo-app/app/stack-overflow/columnSpan.demo.ts#L23" target="_blank">
-![screen shot 2016-12-16 at 1 05 45 pm](https://cloud.githubusercontent.com/assets/210413/21274996/6b640f8a-c390-11e6-87ac-ca85eb6c3983.png)
+<img alt="screen shot 2016-12-16 at 1 05 45 pm" src="https://cloud.githubusercontent.com/assets/210413/21274996/6b640f8a-c390-11e6-87ac-ca85eb6c3983.png">
```
Comment (CaerusKaru) [inline docs/documentation/fxFlex-API.md]: This should be `[screen shot 2016-12-16 at 1 05 45 pm](https://github.com/angular/flex-layout/blob/master/src/demo-app/app/stack-overflow/columnSpan.demo.ts#L23)` with no `<a>` tags

**Comment by PowerKiKi** [comment]
All comments have been fixed.

**Comment by PowerKiKi** [comment]
:+1:

**Comment by ThomasBurleson** [comment]
@PowerKiKi - plz squash your commits. Also, plz add a description in commit body that discusses why this change was necessary.

**Comment by PowerKiKi** [comment]
done

**Pull request merged**